.eslintrc.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true
+  },
+  extends: ['airbnb-base', 'prettier'],
+  globals: {
+    Atomics: 'readonly',
+    SharedArrayBuffer: 'readonly'
+  },
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module'
+  },
+  plugins: ['@typescript-eslint', 'prettier'],
+  rules: {
+    'prettier/prettier': ['error'],
+    'no-unused-vars': 'off',
```

Comment:
I'll include with my update to micro-api once this is published.

Reply:
@ujwal-setlur bringing back up this comment, actually we should enforce `no-unused-vars`  for the majority of our repositories. Since this is a public package it's a little different, the modules are just exported and not used. It's up to the consumer to use them which is why I included an exception for no-unused-vars here.